packages/core/test/types.test.ts

```diff
@@ -76,7 +77,7 @@ describe('StateSchema', () => {
             }
           },
           stop: {
-            always: { target: 'green' }
+            always: { target: '#green' }
```

Comment:
This is ~unrelated~ tangentially related to this PR - without eager transition resolution, this will not throw an error (which it should, since `'green'` is not a child state of `'red'`).

Reply:
so what has been happening here before and how has this been changed by this PR?

Reply:
The target is wrong.

Reply:
Ah, ok 👌